common/changes/@microsoft/rush/fix-tty_2023-09-19-10-01.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@microsoft/rush",
+      "comment": "Fix pnpm's install status printing when pnpm custom tips are defined.",
```

Comment:
@iclanton let's get this merged+released asap. The regression is conspicuous and visually annoying.